eit_dash/main.py

```diff
@@ -1,11 +1,11 @@
 import dash_bootstrap_components as dbc
 from dash import html, page_container
 
-import callbacks.load_callbacks  # noqa
-from app import app
+import eit_dash.callbacks.load_callbacks  # noqa
+from eit_dash.app import app
```

Comment:
I agree. I had to change them again because it was not working otherwise when in the e2e tests, where we are importing the main. We can keep it like this, I think?